ccd-definition/CaseEventToComplexTypes/manageDocs/manageDocuments.json

```diff
@@ -42,18 +42,28 @@
     "ID": "ManageDocuments",
     "CaseEventID": "manageDocuments",
     "CaseFieldID": "supportingEvidenceDocumentsTemp",
-    "ListElementCode": "confidentialWarningLabel",
+    "ListElementCode": "type",
     "FieldDisplayOrder": 5,
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageDocument.type=\"FURTHER_EVIDENCE_DOCUMENTS\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageDocuments",
+    "CaseEventID": "manageDocuments",
+    "CaseFieldID": "supportingEvidenceDocumentsTemp",
+    "ListElementCode": "confidentialWarningLabel",
+    "FieldDisplayOrder": 6,
     "DisplayContext": "READONLY",
-    "EventElementLabel": "<div class='govuk-warning-text'><span class='govuk-warning-text__icon' aria-hidden='true'>!</span><strong class='govuk-warning-text__text'>Check if documents are confidential</strong></div><br>"
+    "EventElementLabel": "<br /><div class='govuk-warning-text'><span class='govuk-warning-text__icon' aria-hidden='true'>!</span><strong class='govuk-warning-text__text'>Check if documents are confidential</strong></div><br>"
```

Comment:
it is preferable to use css instead of new line. You can add e.g. govuk-!-margin-top-8 class to existing div. The only issue could be that ccd has max length for label so it may cause issue. If this is an issue then br is fine

Reply:
problem is label field can only be 200 characters long, import will fail with:

ERROR: value too long for type character varying(200))

so not enough characters to specify css